dockerfile/cuda11.1.1.dockerfile

```diff
@@ -7,7 +7,7 @@ FROM nvcr.io/nvidia/pytorch:20.12-py3
 # NVIDIA:
 #   - CUDA: 11.1.1
 #   - cuDNN: 8.0.5
-#   - NCCL: bootstrap_tag
+#   - NCCL: v2.10.3-1
```

Comment:
If we change the default NCCL version, all baseline need to be re-tested. @abuccts